modules/ROOT/partials/menu-item-ids/mergetags-menu-items.adoc

```diff
@@ -0,0 +1,5 @@
+[cols="1,1,2",options="header"]
+|===
+|Menu item identifier |xref:menus-configuration-options.adoc#example-the-tinymce-default-menu-items[Default Menu Location] |Description
+|`+mergetags+` |Insert |Inserts a merge tag from a nested menu. The nested menu contains the merge tags list, as specified in xref:#mergetags_list[`+mergetags_list+`]
```

Comment:
This xref won't work when included on multiple pages which is will need to be. In general it should only ever be relative like this when the xref target is on the current page.

Reply:
fixed